test/client/fixtures/sandbox/node/dom-processor-test.js

```diff
@@ -282,21 +293,24 @@ test('clean up stylesheet', function () {
 });
 
 test('stylesheet after innerHTML', function () {
-    var div   = $('<div>').appendTo('body')[0];
-    var style = $('<style>')[0];
+    var div   = nativeMethods.createElement.call(document, 'div');
+    var style = nativeMethods.createElement.call(document, 'style');
+
+    nativeMethods.appendChild.call(document.body, style);
+
     var check = function (cssText) {
         strictEqual(cssText.indexOf(styleProcessor.IS_STYLESHEET_PROCESSED_COMMENT), 0);
         strictEqual(cssText.indexOf(styleProcessor.IS_STYLESHEET_PROCESSED_COMMENT, 1), -1);
-        strictEqual(cssText.replace(/^[\s\S]+url\(([\s\S]+)\)[\s\S]+$/, '$1'), urlUtils.getProxyUrl('http://test.ru'));
+        strictEqual(cssText.replace(/^[\s\S]+url\(([\s\S]+)\)[\s\S]+$/, '$1'), urlUtils.getProxyUrl('http://domain.com'));
     };
 
-    eval(processScript('div.innerHTML = "<style>.rule { background: url(http://test.ru) }</style>";'));
+    eval(processScript('div.innerHTML = "<style>.rule { background: url(http://domain.com) }</style>";'));
```

Comment:
I suppose we should use `example.com` instead `domain.com` in this case. `example.com` is created for this purposes